pytest_html/plugin.py

```diff
@@ -157,24 +158,24 @@ def create_asset(self, content, extra_index,
         def append_extra_html(self, extra, extra_index, test_index):
             href = None
             if extra.get('format') == extras.FORMAT_IMAGE:
-                if self.self_contained:
+                content = extra.get('content')
+                if re.match('^file|^http', content):
```

Comment:
Haha, I started with that but thought that re was better looking :P 
However, with the abs/rel path change, I'll revert back to startswith 👍